### PR TITLE
fix(runtime): add syntax file for query

### PR DESCRIPTION
**Pull request by clason**
followup to #17905: also use Lisp syntax for treesitter queries

### CODE REVIEW

File: runtime/syntax/query.lua [hunk 1]
```diff
@@ -0,0 +1,6 @@
+-- Neovim indent file
```
Comment (zeertzjq) [inline runtime/syntax/query.lua:1]:
```suggestion
-- Neovim syntax file
```

Comment (clason) [reply runtime/syntax/query.lua]: copy-pasta 🤦

**Comment by zeertzjq** [comment]
Hmm, I think only very few Lisp keywords are treesitter query keywords, which make the highlighting look a bit strange:
Screenshot_20220413_183542

**Comment by clason** [comment]
It think it looks fine? Certainly better than no highlighting at all (especially for comments)... It's mostly meant as a fallback for people who don't have tree-sitter activated.

(The alternative would be to write a custom syntax file, and that is certainly not worth the effort.)

**Review by zeertzjq — APPROVED** [review]

**Review by vigoux — APPROVED** [review]

**Pull request merged**